ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_multiple_callbacks/README.rst

```diff
@@ -0,0 +1,44 @@
+
+Multiple callbacks example.
+===========================
+
+This example shows a ``Span`` created for a top-level operation, covering a set of asynchronous operations (representing callbacks), and have this ``Span`` finished when **all** of them have been executed.
+
+``Client.send()`` is used to create a new asynchronous operation (callback), and in turn every operation both restores the active ``Span``\ , and creates a child ``Span`` (useful for measuring the performance of each callback).
+
+Implementation details:
+
+
+* For ``threading``, a thread-safe counter is put in each ``Span`` to keep track of the pending callbacks, and call ``Span.finish()`` when the count becomes 0.
+* For ``tornado`` and ``asyncio`` the children corotuines representing the subtasks are simply yielded over, so no counter is needed.
```

Comment:
if this is the case, do we need this test? or is it just for an example?

Reply:
This comment is telling an implementation detail. 

The threads case requires an atomic counter to track the list of active spans in order to end the parent span when all the children ones are finished, i.e. all the child threads are done. 

For the asyncio case it is not needed because the parent span will be ended when the children coroutines are done, i.e. the `await asyncio.gather(*tasks)` finishes.